app/views/courses/show.html.erb

```diff
@@ -8,6 +8,61 @@ var note_upload_url = "<%= url_for(controller:'courses', action:'updateNotes') %
 <%= javascript_include_tag "dropzone.js" %>
 <%= render 'header', course: "active", edit: "", students: "", quiz: "", assignments: "" %>
```

Comment:
The UI glitch you're getting is because of this. We moved the navbar into a partial, so it's all handled by this line now. If you remove lines 13-19, that should fix things.